web/src/pages/about.tsx

```diff
@@ -1,12 +1,37 @@
 import { Box, Button, Grid, Heading, Label } from '@theme-ui/components';
 import { navigate } from 'gatsby';
-import Img from 'gatsby-image';
 import React from 'react';
 import { AboutList } from '../components/about-page/about-list';
 import { Container } from '../components/container';
 import { Layout } from '../components/layouts';
 
-// TODO: fix image showing when Sanity will be done
+const boxesData = [
```

Comment:
I talk about this in #108 as well.

It is relevant here as well, IMHO. 

re https://github.com/bjerkio/website/pull/108#discussion_r600901491